server/models/schema.sql

```diff
@@ -316,51 +315,51 @@ CREATE TABLE `consumption_service` (
   FOREIGN KEY (`service_id`) REFERENCES `service` (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+DROP TABLE IF EXISTS `fee_center`;
 
-DROP TABLE IF EXISTS `cost_center`;
-
-CREATE TABLE `cost_center` (
-  `project_id` smallint(5) unsigned NOT NULL,
+CREATE TABLE `fee_center` (
   `id` smallint(6) NOT NULL AUTO_INCREMENT,
-  `text` varchar(100) NOT NULL,
-  `note` text,
+  `project_id` smallint(5) unsigned NOT NULL,
+  `label` varchar(100) NOT NULL,
+  `is_cost` tinyint(1) DEFAULT 0,
   `is_principal` tinyint(1) DEFAULT 0,
+  `note` text,
   PRIMARY KEY (`id`),
-  UNIQUE KEY `const_center_1` (`text`),
+  UNIQUE KEY `fee_center_1` (`label`),
   KEY `project_id` (`project_id`),
-  FOREIGN KEY (`project_id`) REFERENCES `project` (`id`)
+  FOREIGN KEY (`project_id`) REFERENCES `project` (`id`),
+  INDEX `fee_center_index_1` (`is_cost`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
-DROP TABLE IF EXISTS `cost_center_assignation`;
+DROP TABLE IF EXISTS `cost_assignation`;
 
-CREATE TABLE `cost_center_assignation` (
-  `project_id` smallint(5) unsigned NOT NULL,
+CREATE TABLE `cost_assignation` (
   `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
-  `auxi_cc_id` smallint(6) NOT NULL,
+  `auxiliary_cost_id` smallint(6) NOT NULL,
   `cost` float DEFAULT 0,
-  `date` date DEFAULT NULL,
+  `period_id` mediumint(8) UNSIGNED NOT NULL,
+  `date` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
   `note` text,
   PRIMARY KEY (`id`),
-  KEY `project_id` (`project_id`),
-  KEY `auxi_cc_id` (`auxi_cc_id`),
-  FOREIGN KEY (`project_id`) REFERENCES `project` (`id`),
-  FOREIGN KEY (`auxi_cc_id`) REFERENCES `cost_center` (`id`)
+  UNIQUE KEY `cost_assignation_1` (`auxiliary_cost_id`, `period_id`),
+  KEY `auxiliary_cost_id` (`auxiliary_cost_id`),
+  KEY `period_id` (`period_id`),
+  FOREIGN KEY (`auxiliary_cost_id`) REFERENCES `fee_center` (`id`),
```

Comment:
There is only auxiliary cost center and not auxiliary profit center, I will propose an issue to remove this concept of cost and profit. so we will keep only fee center that will help to restructure this table also.